(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[954],{1706:function(e,t,i){"use strict";i.d(t,{$:function(){return r}});var s=i(5893),a=(i(7294),i(1664)),n="/_next/static/image/config/logo/wur_logo_small.d8950f348b1906c131ed6f1342d0a11b.png",r=function(){return(0,s.jsxs)("div",{className:"footer",children:[(0,s.jsx)("div",{className:"content-border-footer"}),(0,s.jsx)("div",{className:"container text-muted text-center pt-2 px-5",children:(0,s.jsxs)("small",{children:["\xa9 2022 \xa0",(0,s.jsx)(a.default,{href:"https://github.com/vdhooftcompmet/",children:(0,s.jsx)("a",{children:"Van der Hooft Computational Metabolomics Group"})})]})}),(0,s.jsxs)("div",{className:"container text-muted text-center px-5",children:[(0,s.jsxs)("small",{children:["Embedded in the \xa0",(0,s.jsx)(a.default,{href:"https://www.wur.nl/en/Research-Results/Chair-groups/Plant-Sciences/Bioinformatics.htm",children:"Bioinformatics Group"}),"\xa0 of the Plant Sciences Group at WUR"]}),"\xa0",(0,s.jsx)(a.default,{href:"https://www.wur.nl/",children:(0,s.jsx)("img",{className:"wur_footer_logo",src:n,alt:"wur_logo_small"})})]})]})}},7742:function(e,t,i){"use strict";i.r(t),i.d(t,{descriptionItem:function(){return c},default:function(){return l}});var s=i(5893),a=(i(7294),i(1664)),n=i(1706),r=i(7625),o=i(1436);function c(e,t,i){return(0,s.jsxs)("div",{className:"description",children:[(0,s.jsx)("div",{children:(0,s.jsxs)("strong",{children:[(0,s.jsx)(a.default,{href:e,children:t}),"\xa0",(0,s.jsx)(r.G,{icon:o.Xjp,size:"xs"})]})}),i]})}function l(){return(0,s.jsxs)("div",{className:"page-container",children:[(0,s.jsx)("div",{className:"content-wrap",children:(0,s.jsxs)("div",{className:"research-pillar-item",children:[(0,s.jsx)("div",{className:"my-4",children:(0,s.jsx)("div",{className:"text-muted",children:(0,s.jsx)(a.default,{href:"../#about",children:(0,s.jsx)("a",{children:"Main..."})})})}),(0,s.jsx)("div",{className:"research-pillar-card",children:(0,s.jsxs)("div",{className:"research-pillar-message",children:[(0,s.jsx)("a",{className:"title",children:"Linked omics and activity profiles to gain functional insights"}),(0,s.jsx)("br",{}),(0,s.jsx)("a",{className:"subtitle",children:"Research pillar III"}),(0,s.jsx)("br",{}),(0,s.jsx)("br",{}),(0,s.jsx)("div",{children:"In this pillar, we develop and apply integrative omics mining approaches that rely on multi-omics experiments. Supported by the Netherlands eScience Centre with the iOMEGA and NPLinker 2.0 grants, our group and close collaborators work on ways to link metabolomics profiles to information retrieved from other omics disciplines. For example, can we link mass spectra to biosynthetic gene clusters that are predicted from genome sequences using computational genomics (genome mining) workflows? Can we use proteomics or transcriptomics to link gene expression data to metabolite abundance? Integrative omics approaches have the potential to accelerate natural product discovery by brining together complementary structural information about specialized metabolites (thus facilitating their structural elucidation) and we work on challenges at the data level (i.e., launch of the Paired Omics Data Platform to record paired omics datasets in a standardized and machine-readable manner), and develop and apply tools to link genomics and metabolomics data at the structure and substructure levels."}),(0,s.jsx)("br",{}),(0,s.jsx)("h5",{children:"Our contributions to this challenging and exciting field"}),(0,s.jsx)("br",{}),c("https://journals.asm.org/doi/10.1128/mSystems.00726-21","Comprehensive Large-Scale Integrative Analysis of Omics Data To Accelerate Specialized Metabolite Discovery","This perspective paper describes our vision on how integrative omics analyses based on multi-omics experiments can revolutionise microbiome research."),(0,s.jsx)("br",{}),c("https://www.nature.com/articles/s41589-020-00724-z","A community resource for paired genomic and metabolomic data mining","A platform that records paired omics datasets, (meta)genome sequences and mass spectrometry fragmentation profiles, to allow for the development of algorithms to link biosynthetic gene clusters to metabolite spectra to facilitate the elucidation of their structures and functions."),(0,s.jsx)("br",{}),c("https://pubs.rsc.org/en/content/articlelanding/2020/CS/D0CS00162G#!divAbstract","Linking genomics and metabolomics to chart specialized metabolic diversity","Tutorial review that describes current and envisioned workflows to combine genome mining and metabolome mining workflows that rely on paired omics datasets.")]})})]})}),(0,s.jsx)(n.$,{})]})}},279:function(e,t,i){(window.__NEXT_P=window.__NEXT_P||[]).push(["/research_pillar_3",function(){return i(7742)}])}},function(e){e.O(0,[774,523,543,888,179],(function(){return t=279,e(e.s=t);var t}));var t=e.O();_N_E=t}]);