<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><meta name="next-head-count" content="2"/><link rel="preload" href="/_next/static/css/824181c9dfc7266e5b93.css" as="style"/><link rel="stylesheet" href="/_next/static/css/824181c9dfc7266e5b93.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/_next/static/chunks/polyfills-381dbb3c33243b4920e6.js"></script><script src="/_next/static/chunks/webpack-deccef4b6f952a287e77.js" defer=""></script><script src="/_next/static/chunks/framework-6f55793fcf98195a1ae2.js" defer=""></script><script src="/_next/static/chunks/main-4a66f78da9b6ddd46e04.js" defer=""></script><script src="/_next/static/chunks/pages/_app-3af4f57321935ec5d19f.js" defer=""></script><script src="/_next/static/chunks/cb1608f2-adddc8e540181702ff0b.js" defer=""></script><script src="/_next/static/chunks/3543-dafbb5b31716579d343c.js" defer=""></script><script src="/_next/static/chunks/pages/people/Kevin_Mildau_as_postdoc-f8e835c6a2542383a008.js" defer=""></script><script src="/_next/static/4EdMMS9HfDpzPiD78vURN/_buildManifest.js" defer=""></script><script src="/_next/static/4EdMMS9HfDpzPiD78vURN/_ssgManifest.js" defer=""></script></head><body><div id="__next"><div class="page-container"><div class="content-wrap"><div class="people-item"><div class="my-4"><div class="text-muted"><a href="../#people">Main...</a></div></div><div class="people-card"><div class="people"><div class="profile text-center"><img class="img-fluid" src="/_next/static/image/config/people/Kevin_Mildau_as_postdoc_profile.333ee565bbc32a6c33d2c94badd80a59.png" alt="Kevin Mildau as postdoc"/></div><a class="name">Kevin Mildau as postdoc</a><br/><a class="role">Postdoctoral researcher</a><br/><br/><a class="description">Kevin holds a BSc. in Biotechnology (Systems and Synthetic Biology) and a MSc. in Bioinformatics, both from Wageningen University &amp; Research (WUR). After his MSc he continued with a research project on 16S microbiome community statistics at the Mathematical and Statistical Methods group (Biometris, WUR) and Danone Nutricia Research. From there, he continued with a PraeDoc research position at the Department of Analytical Chemistry at University of Vienna under the supervision of Christoph Büschl, Jürgen Zanghellini, and Justin van der Hooft, focusing on computational tool development for untargeted metabolomics. During this PhD project, Kevin developed tools to improve untargeted metabolomics workflows, most notably in the form of homologueDiscoverer, a data mining tool for grouping LC-MS peaks into homologous series in R, and specXplore, an interactive and adjustable mass spectral similarity network exploration dashboard that facilitates data exploration and mass spectral annotation efforts. During his PhD he also started work on msFeaST, a feature-set-testing extension of specXplore.<br/><br/>During his PostDoc at the van der Hooft group Kevin continues his research development efforts by improving the accessibility of specXplore and msFeaST (in development) to the research community and extending their respective functionalities.<br/><br/><br/><div class="mb-4"><h5>Interests</h5></div>- Metabolomics and Cheminformatics<br/>- Mass Spectral Similarity Networking<br/>- Interactive Data Visualization<br/>- Data Science &amp; Engineering<br/>- Statistics &amp; Machine Learning<br/>- Research Software Engineering<br/><br/><br/><div class="mb-4"><h5>Publications</h5></div>Pairwise ratio-based differential abundance analysis of infant microbiome 16S sequencing data, NAR Genomics and Bioinformatics, 2023, <strong><a href="https://doi.org/10.1093/nargab/lqad001">https://doi.org/10.1093/nargab/lqad001</a></strong> <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" class="svg-inline--fa fa-external-link-alt fa-w-16 fa-xs " role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"></path></svg><br/><br/>Homologue series detection and management in LC-MS data with homologueDiscoverer, 2022, Bioinformatics, <strong><a href="https://doi.org/10.1093/bioinformatics/btac647">https://doi.org/10.1093/bioinformatics/btac647</a></strong> <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" class="svg-inline--fa fa-external-link-alt fa-w-16 fa-xs " role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"></path></svg><br/><br/>Good practices and recommendations for using and benchmarking computational metabolomics metabolite annotation tools, 2022, Metabolomics, <strong><a href="https://doi.org/10.1007/s11306-022-01963-y">https://doi.org/10.1007/s11306-022-01963-y</a></strong> <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" class="svg-inline--fa fa-external-link-alt fa-w-16 fa-xs " role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"></path></svg><br/><br/>Preprint: Tailored mass spectral data exploration using the specXplore interactive dashboard, <strong><a href="https://doi.org/10.1101/2023.10.03.560677">https://doi.org/10.1101/2023.10.03.560677</a></strong> <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" class="svg-inline--fa fa-external-link-alt fa-w-16 fa-xs " role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"></path></svg><br/><br/></a><br/></div></div></div></div><div class="footer"><div class="content-border-footer"></div><div class="container text-muted text-center pt-2 px-5"><small>© 2022  <a href="https://github.com/vdhooftcompmet/">Van der Hooft Computational Metabolomics Group</a></small></div><div class="container text-muted text-center px-5"><small>Embedded in the  <a href="https://www.wur.nl/en/Research-Results/Chair-groups/Plant-Sciences/Bioinformatics.htm">Bioinformatics Group</a>  of the Plant Sciences Group at WUR</small> <img class="wur_footer_logo" src="/_next/static/image/config/logo/wur_logo_small.d8950f348b1906c131ed6f1342d0a11b.png" alt="wur_logo_small"/></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{}},"page":"/people/Kevin_Mildau_as_postdoc","query":{},"buildId":"4EdMMS9HfDpzPiD78vURN","nextExport":true,"autoExport":true,"isFallback":false,"scriptLoader":[]}</script></body></html>