(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[7051],{1706:function(e,s,r){"use strict";r.d(s,{$:function(){return o}});var t=r(5893),n=(r(7294),r(1664)),i="/_next/static/image/config/logo/wur_logo_small.d8950f348b1906c131ed6f1342d0a11b.png",o=function(){return(0,t.jsxs)("div",{className:"footer",children:[(0,t.jsx)("div",{className:"content-border-footer"}),(0,t.jsx)("div",{className:"container text-muted text-center pt-2 px-5",children:(0,t.jsxs)("small",{children:["\xa9 2022 \xa0",(0,t.jsx)(n.default,{href:"https://github.com/vdhooftcompmet/",children:(0,t.jsx)("a",{children:"Van der Hooft Computational Metabolomics Group"})})]})}),(0,t.jsxs)("div",{className:"container text-muted text-center px-5",children:[(0,t.jsxs)("small",{children:["Embedded in the \xa0",(0,t.jsx)(n.default,{href:"https://www.wur.nl/en/Research-Results/Chair-groups/Plant-Sciences/Bioinformatics.htm",children:"Bioinformatics Group"}),"\xa0 of the Plant Sciences Group at WUR"]}),"\xa0",(0,t.jsx)(n.default,{href:"https://www.wur.nl/",children:(0,t.jsx)("img",{className:"wur_footer_logo",src:i,alt:"wur_logo_small"})})]})]})}},8639:function(e,s,r){"use strict";r.r(s),r.d(s,{default:function(){return c}});var t=r(5893),n=(r(7294),r(1664)),i=r(1706),o=r(7625),a=r(1436);function c(){return(0,t.jsxs)("div",{className:"page-container",children:[(0,t.jsx)("div",{className:"content-wrap",children:(0,t.jsxs)("div",{className:"news-item",children:[(0,t.jsxs)("div",{className:"my-4",children:[(0,t.jsx)("div",{className:"text-muted",children:(0,t.jsx)(n.default,{href:"../#news",children:(0,t.jsx)("a",{children:"Main..."})})}),(0,t.jsx)("div",{className:"text-mutedz",children:(0,t.jsx)(n.default,{href:"../archive",children:(0,t.jsx)("a",{children:"Archive..."})})})]}),(0,t.jsx)("div",{className:"message-card",children:(0,t.jsxs)("div",{className:"message",children:[(0,t.jsx)("a",{className:"title",children:"Preprint out on new version NPOmix"}),(0,t.jsx)("br",{}),(0,t.jsx)("a",{className:"date",children:"2022-07-22"}),(0,t.jsx)("br",{}),(0,t.jsx)("br",{}),(0,t.jsxs)("a",{className:"description",children:[(0,t.jsx)("strong",{children:(0,t.jsx)(n.default,{href:"../people/Joris_Louwen",children:"Joris"})})," and ",(0,t.jsx)("strong",{children:(0,t.jsx)(n.default,{href:"../people/Justin_van_der_Hooft",children:"Justin"})}),", together with Tiago Leao, ",(0,t.jsx)("strong",{children:(0,t.jsx)(n.default,{href:"../collabs/The_Dorrestein_Laboratory",children:"Pieter Dorrestein"})}),", and collaborators, worked on an updated version of NPOmix, a fingerprint-based tool to link gene clusters to mass spectra. Integrated omics mining has been coined as a very promising tool for natural product discovery; yet, it is very challenging.",(0,t.jsx)("br",{}),(0,t.jsx)("br",{}),"NPOmix is a machine learning based tool that uses fingerprints to find links between the two omics data types of genomics and metabolomics. The tool can facilitate natural product discovery by proposing gene clusters that are linked to annotated mass spectra (i.e., orphan molecules) to find their biosynthetic origin, or vice versa: to find the mass spectra of products to decipher the molecular products of predicted biosynthetic gene clusters.",(0,t.jsx)("br",{}),(0,t.jsx)("br",{}),"You can find out more in the ",(0,t.jsx)("strong",{children:(0,t.jsx)(n.default,{href:"https://www.biorxiv.org/content/10.1101/2021.10.05.463235v2",children:"preprint"})})," ",(0,t.jsx)(o.G,{icon:a.Xjp,size:"xs"}),".",(0,t.jsx)("br",{}),(0,t.jsx)("br",{}),"Do you want to know more about how our group works on linking omics data? Please find out more ",(0,t.jsx)("strong",{children:(0,t.jsx)(n.default,{href:"../research_pillar_3",children:"here"})}),".",(0,t.jsx)("br",{}),(0,t.jsx)("br",{})]}),(0,t.jsx)("br",{})]})})]})}),(0,t.jsx)(i.$,{})]})}},7818:function(e,s,r){(window.__NEXT_P=window.__NEXT_P||[]).push(["/news/20220722_Preprint_out_on_new_version_NPOmix",function(){return r(8639)}])}},function(e){e.O(0,[9774,8523,3543,2888,179],(function(){return s=7818,e(e.s=s);var s}));var s=e.O();_N_E=s}]);